StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetLoader.swift

```diff
@@ -71,7 +71,7 @@ final class PaymentSheetLoader {
                     }
 
                 STPAnalyticsClient.sharedClient.logPaymentSheetEvent(event: .paymentSheetLoadSucceeded,
-                                                                     duration: loadingStartDate.timeIntervalSinceNow)
+                                                                     duration: Date().timeIntervalSince(loadingStartDate))
```

Comment:
If we change this from negative to positive, i'm guessing there's some way to unify the negative times with the positive ones we will be sending?